Loop/View Controllers/CarbAbsorptionViewController.swift

```diff
@@ -48,7 +48,7 @@ final class CarbAbsorptionViewController: LoopChartsTableViewController, Identif
                     self?.reloadData(animated: true)
                 }
             },
-            notificationCenter.addObserver(forName: .HKUserPreferencesDidChange, object: deviceManager.loopManager.glucoseStore.healthStore, queue: nil) {[weak self] _ in
+            notificationCenter.addObserver(forName: .HKUserPreferencesDidChange, object: deviceManager.glucoseStore.healthStore, queue: nil) {[weak self] _ in
```

Comment:
Going through the device data manager to get to the stores

Reply:
This is a good cleanup; there are a bunch more that can be cleaned up in this same manner.

Reply:
Probably easiest to find by making the store vars on LDM private. <- and that's probably a good change to keep.

Reply:
Wow, there were like 50 references